runtime/drivers/file/model_executor_olap_self.go

```diff
@@ -6,20 +6,24 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"io"
 	"os"
 	"time"
 
 	"github.com/apache/arrow/go/v14/arrow"
 	"github.com/apache/arrow/go/v14/arrow/array"
 	"github.com/apache/arrow/go/v14/arrow/memory"
 	"github.com/apache/arrow/go/v14/parquet/pqarrow"
+	"github.com/c2h5oh/datasize"
 	"github.com/mitchellh/mapstructure"
 	runtimev1 "github.com/rilldata/rill/proto/gen/rill/runtime/v1"
 	"github.com/rilldata/rill/runtime/drivers"
 	"github.com/rilldata/rill/runtime/pkg/jsonval"
 	"github.com/xuri/excelize/v2"
 )
 
+const maxParquetRowGroupLen = 4 * 1024 * 1024
```

Comment:
The Parquet docs recommend 512 MB, so this is probably too small: https://parquet.apache.org/docs/file-format/configurations/

Reply:
The underlying writer uses a new row group if the number of rows is higher than 64M. I reduced it by a factor of 16 to 4M. 
The default for clickhouse exports is 1M and that for duckdb is just 122,880.

Reply:
Got it, so this is the number of rows, not bytes